### DISCUSSION TITLE
2023년 04월 01일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#25] [General]
# 2023년 04월 01일

[ 2023-04-01 ]

https://cyma-s.github.io/2023-04-01/

**Comment by Choi-JJunho** [comment]
감정을 배우는 것도 TIL이죠~
베로 잘하고있어요 👍

**Reply by Cyma-s** [reply]
선플맨 주노 방학에도 열일하네 🚀

**Comment by greeng00se** [comment]
고민을 하는 부분에서 이미 좋은 선생님이군요 👍

**Comment by be-student** [comment]
역시 좋은  선생님 
:good
